SwiftSnake/SwiftSnake/SKNodes/SnakeHeadNode.swift

```diff
@@ -9,6 +9,22 @@
 import Foundation
 import SpriteKit
 
-class SnakeHeadNode: SKNode {
+enum SnakeHeadDirection: String {
+    case Up = "SnakeHeadUp",
+    Down = "SnakeHeadDown",
+    Right = "SnakeHeadRight",
+    Left = "SnakeHeadLeft"
+}
+
+class SnakeHeadNode: SKSpriteNode {
+    
+    //MARK:- Factory
+    
+    static func withDirection(direction: SnakeHeadDirection) -> SnakeHeadNode {
+        let node = SnakeHeadNode(imageNamed: direction.rawValue)
```

Comment:
what about using rotation/mirror of one image instead of different images?
that would allow you to animate the head rotation when changing directions.